docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: docs
+title: Getting Started
+permalink: /documentation/getting-started/
+---
+# Getting Started
+
+This guide provides step-by-step instructions on how to create a functional web test with TestCafe and contains the following sections.
+
+* [Installing TestCafe](#installing-testcafe)
+* [Creating a Test](#creating-a-test)
+* [Running the Test](#running-the-test)
+* [Viewing the Test Results](#viewing-the-test-results)
+* [Adding Test Code](#adding-test-code)
+  * [Performing Actions on the Page](#performing-actions-on-the-page)
+  * [Observing the Page State](#observing-the-page-state)
+  * [Assertions](#assertions)
+
+## Installing TestCafe
+
+Make sure that [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) are installed on your computer, then run a single command:
+
+```bash
+npm install --save-dev testcafe
+```
+
+## Creating a Test
+
+To create a test, create a new .js file anywhere on your computer.
+This file must have the special structure: tests must be organized into fixtures. So, first you need to declare a fixture by using the [fixture](../test-api/test-code-structure.md#fixtures) function.
+
+You can optionally specify a start page URL for the fixture by using the [page](../test-api/test-code-structure.md#specifying-the-start-webpage) function.
+In this tutorial, you will create a test for the [http://testcafe.devexpress.com/example](http://testcafe.devexpress.com/example) sample page.
+
+Then create the [test](/test-api/test-code-structure.md#tests) function where you will further place test code.
+
+```js
+fixture `Getting Started`
+    .page('http://testcafe.devexpress.com/example');
+
+test('MyTest', async t => {
+    // Test code
+});
+```
+
+## Running the Test
+
+You can simply run the test from a command shell, by calling a single command where you specify the [target browsers](../using-testcafe/command-line-interface.md#browser-list) and [file path](../using-testcafe/command-line-interface.md#file-pathglob-pattern).
+
+```bash
+testcafe ie,chrome mytest.js
+```
+
+TestCafe will automatically open the chosen browsers and start the test execution within them simultaneously.
+
+For more information on how to configure the test run, see [Command Line Interface](../using-testcafe/command-line-interface.md).
+
+## Viewing the Test Results
+
+While the test is running, TestCafe is gathering information about the test run and outputing the report right into a command shell.
+
+IMAGE
+
+For more information, see [Reporters](../using-testcafe/common-concepts/reporters.md).
+
+## Adding Test Code
```

Comment:
> Moreover, the whole flow gets strange

It will also be strange if we name this section "Creating Basic Test" or "Implementing Test Logic".

> What if we get rid of this section and place "Performing Actions on a Page" to the same level as "Viewing results"?

I don't like this idea.